decidim-surveys/app/forms/decidim/surveys/admin/survey_question_form.rb

```diff
@@ -9,11 +9,19 @@ class SurveyQuestionForm < Decidim::Form
         attribute :id, String
         attribute :position, Integer
         attribute :mandatory, Boolean, default: false
+        attribute :question_type, String
+        attribute :answer_options, Array[SurveyQuestionAnswerOptionForm]
         attribute :deleted, Boolean, default: false
 
         translatable_attribute :body, String
 
         validates :position, numericality: { greater_than_or_equal_to: 0 }
+        validates :question_type, inclusion: { in: SurveyQuestion::TYPES }
+
+        # Private: answer options should be mapped manually.
+        def map_model(model)
+          self.answer_options = model.answer_options
```

Comment:
Doesn't it call the method? I mean, you are not changing anything here. We usually use this method because of some form field names not matching the model method, so we need to manually set them. Here you are not doing anything special, you're calling the same method name that you are setting so it should be done automatically :/